src/woodbury_pd_mat.jl

```diff
@@ -88,3 +88,7 @@ end
 # NOTE: the parameterisation to scale up the Woodbury matrix is not unique. Here we
 # implement one way to scale it.
 *(a::WoodburyPDMat, c::Real) = WoodburyPDMat(a.A, a.D * c, a.S * c)
+*(c::Real, a::WoodburyPDMat) = a * c
```

Comment:
I think this is already implemented for `AbstractPDMat` so concrete PDMat doesn't implement it again

Reply:
Yeah, but this originally led to the confusion about why this was working. I don't see any issue in making this explicit here.